src/GovUk.Education.ExploreEducationStatistics.Notifier.Tests/Services/EmailServiceTests.cs

```diff
@@ -0,0 +1,227 @@
+using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Options;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Services;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Notify.Exceptions;
+using Notify.Interfaces;
+using Notify.Models.Responses;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils.MockUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Notifier.Tests.Services;
+
+public class EmailServiceTests
+{
+    private const string TestApiKeyErrorMessage =
+        "Can't send to this recipient using a team-only API key";
+
+    private static readonly AppOptions DefaultAppOptions = new() { SuppressExceptionsForTeamOnlyApiKeyErrors = false };
+
+    [Fact]
+    public void SendEmail_Success()
+    {
+        var email = "test@test.com";
+        var templateId = "test-template-id";
+        var values = new Dictionary<string, dynamic>
+        {
+            { "test-key", "test-value" },
+            { "test-key-2", 2 }
+        };
+
+        var notificationClient = new Mock<INotificationClient>(MockBehavior.Strict);
+
+        notificationClient
+            .Setup(s =>
+                s.SendEmail(
+                    email,
+                    templateId,
+                    values,
+                    null,
+                    null,
+                    null))
+            .Returns(new EmailNotificationResponse());
+
+        var service = BuildService(
+            notificationClient: notificationClient.Object);
+
+        service.SendEmail(
+            email: email,
+            templateId: templateId,
+            values: values);
+
+        notificationClient.Verify(
+            s => s.SendEmail(
+                email,
+                templateId,
+                values,
+                null,
+                null,
+                null)
+            , Times.Once);
+    }
+
+    [Fact]
+    public void SendEmail_GenericNotifyClientExceptionRaised_RethrowsException()
+    {
+        var email = "test@test.com";
+        var templateId = "test-template-id";
+        var values = new Dictionary<string, dynamic>
+        {
+            { "test-key", "test-value" },
+            { "test-key-2", 2 }
+        };
+
+        var notificationClient = new Mock<INotificationClient>(MockBehavior.Strict);
+
+        var originalException = new NotifyClientException(TestApiKeyErrorMessage);
```

Comment:
I think this should be `"Generic error"` rather than `TestApiKeyErrorMessage` as its in the test `GenericNotifyClientExceptionRaised`.

Reply:
D'oh yeah well spotted!  Swapped thanks.